src/components/invites/InvitesModal.tsx

```diff
@@ -59,12 +83,27 @@ const InvitesModal: FunctionComponent = () => {
         <DialogContentText
           sx={ { backgroundColor: theme.colors.grey500, px: 3 } }
         >
-          <Typography variant="subtitle1">
-            You need to take action on these pending invitations.
-          </Typography>
+          <Stack
+            alignItems={ [null, null, "center"] }
+            columnGap={ 1 }
+            flexDirection={ [null, null, "row"] }
```

Comment:
@scandycuz @dmkirshon Recently i found out that `flexDirection={ [null, null, "row"] }` is the same as doing `sx={{flexDirection:  [null, null, "row"] }}` 🪄